app/components/channel_item/channel_item.tsx

```diff
@@ -120,13 +125,15 @@ const ChannelItem = ({
     isOnCenterBg = false,
     showChannelName = false,
     isOnHome = false,
+    urgentMentionsCount = 0,
 }: Props) => {
     const {formatMessage} = useIntl();
     const theme = useTheme();
     const isTablet = useIsTablet();
     const styles = getStyleSheet(theme);
 
     const channelName = (showChannelName && !isDMorGM(channel)) ? channel.name : '';
+    const hasUrgent = urgentMentionsCount > 0;
```

Comment:
Should isBolden also be set if hasUrgent?

Reply:
The same as the above reply, the mention count is covered in urgent_count, so hasUrgent will only be true if there is hasMentions. So, there is no need to explicitly add it.